projects/media-viewer/src/lib/store/actions/annotations.action.ts

```diff
@@ -0,0 +1,101 @@
+import { Action } from '@ngrx/store';
+import {Annotation} from '../../annotations/annotation-set/annotation-view/annotation.model';
+import {SelectionAnnotation} from '../../annotations/models/event-select.model';
+
+export const LOAD_ANNOTATION_SET = '[Annotations] Load Annotation Set';
+export const LOAD_ANNOTATION_SET_SUCCESS = '[Annotations] Load Annotation Set Success';
+export const LOAD_ANNOTATION_SET_FAIL = '[Annotations] Load Annotation Set Fail';
+
+export const SAVE_ANNOTATION = '[Annotations] Save Annotation';
+export const SAVE_ANNOTATION_SUCCESS = '[Annotations] Save Annotation Success';
+export const SAVE_ANNOTATION_FAIL = '[Annotations] Save Annotation Fail';
+export const ADD_OR_EDIT_COMMENT = '[Annotations] Add or Edit Comment';
+
+export const ADD_PAGE = '[Annotations] Add Page';
+
+export const DELETE_ANNOTATION = '[Annotations] Delete Annotation';
+export const DELETE_ANNOTATION_SUCCESS = '[Annotations] Delete Annotation Success';
+export const DELETE_ANNOTATION_FAIL = '[Annotations] Delete Annotation Fail';
+
+export const SELECT_ANNOTATION  = '[Annotations] Select Annotation';
+export const SEARCH_COMMENT  = '[Comments] Search Comments';
+
```

Comment:
all these actions have any as payload. might be worth being a bit more specific. will help us catch errors early

Reply:
done